httpbakery/client.go

```diff
@@ -313,6 +313,9 @@ func (c *Client) HandleError(reqURL *url.URL, err error) error {
 		return errgo.Notef(err, "cannot make cookie")
 	}
 	cookie.Path = cookiePath
+	if name := respErr.Info.CookieName; name != "" {
```

Comment:
I would argue that this should be MacaroonName, because apart from anything else it's not the actual cookie name.